services/accounts/accounts_test.go

```diff
@@ -0,0 +1,334 @@
+package accounts
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	empty "github.com/golang/protobuf/ptypes/empty"
+	"github.com/stretchr/testify/mock"
+
+	accountspb "github.com/openbank/gunk/gunk/v1/accounts"
+	typespb "github.com/openbank/gunk/gunk/v1/types"
+	"github.com/openbank/openbank/storage"
+)
+
+func TestHandler_GetAccount(t *testing.T) {
+	type args struct {
+		ctx context.Context
+		req *accountspb.GetAccountRequest
+	}
+	tests := []struct {
+		name      string
+		args      args
+		mockCalls func(m *storage.MockAccountStore)
+		want      *accountspb.Account
+		wantErr   bool
+	}{
+		{
+			name: "get account ok",
+			args: args{
+				ctx: context.Background(),
+				req: &accountspb.GetAccountRequest{
+					AccountID: "1",
+				},
+			},
+			mockCalls: func(m *storage.MockAccountStore) {
+				m.On("GetAccount", mock.Anything, mock.Anything).Return(&accountspb.Account{
+					AccountID: "1",
+				}, nil)
+			},
+			want: &accountspb.Account{
+				AccountID: "1",
+			},
+		},
+		{
+			name: "get account not found",
+			args: args{
+				ctx: context.Background(),
+				req: &accountspb.GetAccountRequest{
+					AccountID: "1",
+				},
+			},
+			mockCalls: func(m *storage.MockAccountStore) {
+				m.On("GetAccount", mock.Anything, mock.Anything).Return(nil, fmt.Errorf("not found"))
+			},
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Arrange
+			s := &storage.MockAccountStore{}
+			tt.mockCalls(s)
+			h := NewHandler(s)
+
+			// Act
+			got, err := h.GetAccount(tt.args.ctx, tt.args.req)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("Handler.GetAccount() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+
+			// Assert
+			if !reflect.DeepEqual(got, tt.want) {
```

Comment:
Have you looked at https://github.com/google/go-cmp for comparison in test @cedric-parisi? I used `reflect.DeepEqual` last time, but generally prefer the `cmp.DeepEqual` now.